store/copr/coprocessor.go

```diff
@@ -70,6 +71,11 @@ type CopClient struct {
 
 // Send builds the request and gets the coprocessor iterator response.
 func (c *CopClient) Send(ctx context.Context, req *kv.Request, variables interface{}, option *kv.ClientSendOption) kv.Response {
+	if config.CheckTableBeforeDrop && !checkKeyRangeSorted(req.KeyRanges) {
```

Comment:
Why we need check CheckTableBeforeDrop?

Reply:
The flag `config.CheckTableBeforeDrop`  is always false, unless you build tidb-server with:

```
WITH_CHECK=1 make server
```

It's really hard to ensure this assumption, but if I always check it, the performance cost is unacceptable.
So do it when build with WITH_CHECK